module-presentation/src/main/kotlin/site/yourevents/s3/dto/response/PreSignedUrlResponse.kt

```diff
@@ -0,0 +1,9 @@
+package site.yourevents.s3.dto.response
+
+data class PreSignedUrlResponse(
+    val preSignedUrl: String
+) {
+    companion object {
+        fun of(preSignedUrl: String) : PreSignedUrlResponse = PreSignedUrlResponse(preSignedUrl)
```

Comment:
매개변수 하나를 통해 생성되기에, 메서드명은 from으로 바꿔주시는게 좋을 것 같습니다!!
```suggestion
        fun from(preSignedUrl: String) : PreSignedUrlResponse = PreSignedUrlResponse(preSignedUrl)
```

Reply:
[해당 자료](https://velog.io/@saint6839/%EC%A0%95%EC%A0%81-%ED%8C%A9%ED%86%A0%EB%A6%AC-%EB%A9%94%EC%84%9C%EB%93%9C-%EB%84%A4%EC%9D%B4%EB%B0%8D-%EB%B0%A9%EC%8B%9D)를 보시면 좋을 것 같아서 공유드립니다!

Reply:
아 이부분은 처음 알게 된 부분이네요!! 무작정 정완님이 우선 구현한 코드와 최대한 형식을 맞춘다는 생각으로 구현하다보니 문제가 생겼던 것 같습니다. 좋은 지적 감사합니다!!